lib/bemxjst/context.js

```diff
@@ -33,6 +33,10 @@ Context.prototype.xmlEscape = utils.xmlEscape;
 Context.prototype.attrEscape = utils.attrEscape;
 Context.prototype.jsAttrEscape = utils.jsAttrEscape;
 
+Context.prototype.onError = function(ctx, e) {
+  console.error('bem-xjst rendering error:', ctx, e);
```

Comment:
But here it can be `{ block: ctx.block, ... }`

Reply:
I like @zxqfox suggestion